exercises/K-functions-parameters/exercise.js

```diff
@@ -1,9 +1,11 @@
 // Complete the function so that it takes input parameters
-function multiply() {
-  // Calculate the result of the function and return it
+function multiply(a, b) {
+  return a * b;
```

Comment:
Avoid using single letters.  It can be difficult to understand the purpose of the variable. Best to use clear variable names.